tmcprototype/CspSubarrayLeafNode/test/CspSubarrayLeafNode_test.py

```diff
@@ -91,20 +103,20 @@ def test_Reset(self, create_cspsubarray1_proxy):
         #create_cspsubarray1_proxy.device.Reset() is None
         # PROTECTED REGION END #    //  CspSubarrayLeafNode.test_Reset
 
-    def test_ConfigureScan(self, tango_context):
-        """Test for ConfigureScan"""
-        # PROTECTED REGION ID(CspSubarrayLeafNode.test_ConfigureScan) ENABLED START #
-        test_input = '{"csp":{"frequencyBand":"1","delayModelSubscriptionPoint":"",' \
-                     '"visDestinationAddressSubscriptionPoint":"","fsp":[{"fspID":"1","functionMode":' \
-                     '"CORR","frequencySliceID":1,"integrationTime":1400,"corrBandwidth":0,' \
-                     '"channelAveragingMap":[]},{"fspID":"2","functionMode":"CORR","frequencySliceID":1,' \
-                     '"integrationTime":1400,"corrBandwidth":0,"channelAveragingMap":[]}]}}'
-        res = tango_context.device.ConfigureScan(test_input)
-        tango_context.device.status()
-        time.sleep(1)
-        assert CONST.STR_CONFIGURESCAN_SUCCESS in tango_context.device.activityMessage \
-               and res is None
-        # PROTECTED REGION END #    //  CspSubarrayLeafNode.test_ConfigureScan
+    # def test_ConfigureScan(self, tango_context):
```

Comment:
Test case commented can be added with a TODO tag.

Reply:
Done.